kuwala/pipelines/population-density/src/Processor.py

```diff
@@ -46,7 +47,23 @@ def start(files: [dict], output_dir: str):
 
         df = reduce((lambda d1, d2: d1.join(d2, ['h3Index'], 'full').repartition(number_of_partitions, 'h3Index')), dfs)
 
-        df.write.mode('overwrite').parquet(output_dir + 'result.parquet')
+        today=str(dte.today())
+
+        if (os.path.isdir(output_dir + 'result.parquet'+'_'+today)):
+            print("parquet files already exist")
+            deci=''
+            while(deci!='n' or deci!='N' or deci!='y' or deci!='Y'):
+                deci=input("do you want to overwrite? [Y] or skip donwload [N]?   ")
+                if deci=='n' or deci=='N':
+                    print("Download skipped.")
+                    break
+                elif deci=='y' or deci=='Y':
+                    df.write.mode('overwrite').parquet(output_dir + 'result.parquet'+'_'+today)
```

Comment:
The date should be added before the parquet extenstion. 
```suggestion
                    df.write.mode('overwrite').parquet(output_dir + 'result_'+today+'.parquet')
```
The same goes for line no 66

Reply:
Ah yes this one is correct, thanks for correcting this.
Also line 52, to follow the change

Reply:
Oh yea, I totally missed that line 52.